src/rgw/rgw_amqp_1.h

```diff
@@ -0,0 +1,71 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*- 
+// vim: ts=8 sw=2 smarttab ft=cpp
```

Comment:
style: please convert tabs to 2 spaces

Reply:
thanks for the reminder. I've substituted all tabs.